learn/getting_started/supported_os.md

```diff
@@ -0,0 +1,25 @@
+# Supported operating systems
+
+Please ensure your operating system is listed on this page before installing the Meilisearch binary. Refer to the [changelog](https://github.com/meilisearch/MeiliSearch/releases) for more information on operating system support changes.
+
+If the provided [binaries](/learn/getting_started/quick_start.md#local-installation) don't work for you, we recommend compiling Meilisearch from the source files. If the compilation fails, Meilisearch is not compatible with your machine.
```

Comment:
Besides the minor edits, I would also recommend moving this paragraph so it's after the supported OS list. This way users get the list ASAP, and then only need to read what to do if something goes wrong/their OS is not listed.

```suggestion
If the provided [binaries](/learn/getting_started/quick_start.md#local-installation) don't work for you, try building Meilisearch from source. If compilation fails, Meilisearch is not compatible with your machine.
```